src/m365/pp/commands/website/website-get.ts

```diff
@@ -0,0 +1,77 @@
+import { Logger } from '../../../../cli/Logger.js';
+import { globalOptionsZod } from '../../../../Command.js';
+import { powerPlatform } from '../../../../utils/powerPlatform.js';
+import { validation } from '../../../../utils/validation.js';
+import { zod } from '../../../../utils/zod.js';
+import PowerPlatformCommand from '../../../base/PowerPlatformCommand.js';
+import commands from '../../commands.js';
+import { z } from 'zod';
+
+const options = globalOptionsZod
+  .extend({
+    url: zod.alias('u', z.string().optional()
+      .refine(url => url === undefined || validation.isValidPowerPagesUrl(url) === true, url => ({
+        message: `'${url}' is not a valid Power Pages URL.`
+      }))
+    ),
+    id: zod.alias('i', z.string().uuid().optional()),
+    name: zod.alias('n', z.string().optional()),
+    environmentName: zod.alias('e', z.string()),
+    asAdmin: z.boolean().default(false)
```

Comment:
This option is not in the specs and the docs. Let's remove it here as well. Unless it has added benefit, in which case this should be discussed in the related issue.